config/sync/ultimate_cron.job.node_revision_delete_cron.yml

```diff
@@ -11,6 +11,9 @@ module: node_revision_delete
 callback: node_revision_delete_cron
 scheduler:
   id: simple
+  configuration:
+    rules:
+      - '* * * * *'
```

Comment:
"tell me more"

Reply:
not til i see if this fixes it. :smile: its not that cron runs every time, its that this task will run on every cron run.

Reply:
That's "more". Thanks 😄